tests/Feature/Api/Auth/ResetPasswordTest.php

```diff
@@ -74,4 +75,51 @@ public function test_incorrect_token()
 
         $response->assertStatus(400);
     }
+
+    public function test_change_password_success()
```

Comment:
We have actingAsToken method in ApiTestCase

Reply:
used authenticate method instead